docs/guides/waitlist.mdx

```diff
@@ -0,0 +1,154 @@
+
+<Steps>
+  ### Install `@clerk/nextjs`
+
+  Clerk's [Next.js SDK](/docs/references/nextjs/overview) gives you access to prebuilt components, React hooks, and helpers to make user authentication easier.
+
+  Run the following command to install the SDK:
+
+  <CodeBlockTabs options={["npm", "yarn", "pnpm"]}>
+    ```bash {{ filename: 'terminal' }}
+    npm install @clerk/nextjs
+    ```
+
+    ```bash {{ filename: 'terminal' }}
+    yarn add @clerk/nextjs
+    ```
+
+    ```bash {{ filename: 'terminal' }}
+    pnpm add @clerk/nextjs
+    ```
+  </CodeBlockTabs>
+
+  ### Set your Clerk API keys
+
+  <If condition={experiment.enabled}>
+    <ExperimentCreateAccountFromDocsQuickstart params={experiment} />
+  </If>
+
+  <If condition={!experiment.enabled}>
+    <SignedIn>
+      Add the following keys to your `.env.local` file. These keys can always be retrieved from the [**API Keys**](https://dashboard.clerk.com/last-active?path=api-keys) page of your Clerk Dashboard.
+    </SignedIn>
+
+    <SignedOut>
+      1. In the Clerk Dashboard, navigate to the [**API Keys**](https://dashboard.clerk.com/last-active?path=api-keys){{ track: 'exp_create_account_nextjs_quickstart' }} page.
+      1. In the **Quick Copy** section, copy your Clerk publishable and secret key.
+      1. Paste your keys into your `.env.local` file.
+
+      The final result should resemble the following:
+    </SignedOut>
+  </If>
+
+  ```env {{ filename: '.env.local' }}
+  NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY={{pub_key}}
+  CLERK_SECRET_KEY={{secret}}
+  ```
+
+  ### Enable Waitlist mode
+
+  To enable **Waitlist** mode, follow these steps:
+
+  1. In the Clerk Dashboard, navigate to the [**Restrictions**](https://dashboard.clerk.com/last-active?path=user-authentication/restrictions) page.
+  1. Under the **Sign-up modes** section, enable **Waitlist**.
+
+  To manage users on your waitlist:
+
+  1. In the Clerk Dashboard, navigate to the [**Waitlist**](https://dashboard.clerk.com/last-active?path=waitlist) page.
+  1. On the right-side of a user's row, select the menu icon (...).
+  1. Select **Invite** to invite the user to your application. Select **Deny** to deny the user access to your application.
+
+  ### Add `<ClerkProvider>` to your app
+
+  The [`<ClerkProvider>`](/docs/components/clerk-provider) component wraps your app to provide active session and user context to Clerk's hooks and other components. To use the `<Waitlist />` component in your app, you must provide the `waitlistUrl` prop.
+
+  ```tsx {{ filename: 'app/layout.tsx', mark: [6] }}
+  import { ClerkProvider } from '@clerk/nextjs'
+  import './globals.css'
+
+  export default function RootLayout({ children }: { children: React.ReactNode }) {
+    return (
+      <ClerkProvider waitlistUrl="/waitlist">
```

Comment:
@alexisintech I saw that you render the `<Waitlist />` component in `/`, so the `waitlistUrl` prop should be `/`

Reply:
fixed! [a93f692](https://github.com/clerk/clerk-docs/pull/1734/commits/a93f692050989406698140b8968afd82161215e8)